metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/MaaSFunctions.java

```diff
@@ -188,15 +189,27 @@ public void initialize(Context context) {
       catch(Exception ex) {
         LOG.error(ex.getMessage(), ex);
       }
+      finally {
+        isInitialized = true;
```

Comment:
I'd prefer for initialization to not continue to retry for every message, but rather the initialization routine should throw an exception.